Project.toml

```
@@ -1,11 +1,11 @@
 name = "NormalizingFlows"
 uuid = "50e4474d-9f12-44b7-af7a-91ab30ff6256"
-version = "0.1.1"
+version = "0.1.2"
```

```suggestion
version = "0.2.0"
```
given that `train_flow` is exported and it's output are changed?